core/node/node_framework/src/implementations/layers/eth_sender/aggregator.rs

```diff
@@ -108,22 +153,29 @@ impl WiringLayer for EthTxAggregatorLayer {
             .map(BoundEthInterface::sender_account);
 
         let config = self.eth_sender_config.sender.context("sender")?;
+        let mut connection = replica_pool
+            .connection_tagged("eth_sender")
+            .await
+            .map_err(DalError::generalize)?;
```

Comment:
This mixes business logic into wiring logic; so e.g. errors related to DB will be reported as wiring errors, and delays in obtaining connection / executing SQL queries will block wiring. Is it possible to move a connection *pool* into the aggregator and read its Merkle tree asynchronously (i.e., after wiring is finished)?

Reply:
I can do make the merkle tree `OnceCell` and then set the value somewhere down the line if this is better